src/services/zapper.ts

```diff
@@ -114,7 +114,7 @@ export class ZapperService extends Service {
       throw new SdkError(`Only Ethereum is supported for token market data, got ${this.chainId}`);
     }
 
-    const url = "https://api.zapper.fi/v1/protocols/yearn/token-market-data";
+    const url = "https://api.zapper.fi/v2/apps/yearn/tokens";
```

Comment:
Do we need to change the `type` parameter to `groupId`?

Reply:
> Do we need to change the `type` parameter to `groupId`?

Yes! Good catch, had that in Insomnia but forgot to update in the `params`, done with https://github.com/yearn/yearn-sdk/pull/284/commits/4d6b11ab96b67155f849abd49b4f988b1aa3fda7 -- test updated with https://github.com/yearn/yearn-sdk/pull/284/commits/9c8b0d2320dbd0c17a91b8c267fabab1aff94496

<img width="2352" alt="Screen Shot 2022-05-10 at 10 03 22 am" src="https://user-images.githubusercontent.com/78794805/167517385-45d9d63b-7921-4aae-925e-f5dd0727e54c.png">

Reply:
Double checked and seems ok now, good catch @jstashh.

Noticed the response interface `VaultTokenMarketData` is no longer the same. Since we just need the `address` property from the response, maybe we should eliminate the rest from the interface. Will leave it to you if done here or when Zod gets added :)

Reply:
I'd say in Zod since as you said we only need the `address` and that's still the same, but we're indeed casting it incorrectly.